index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width"> <!-- чтобы @media работал в хроме при нажатом Toggle Device Toolbar-->
+    <title>Responsive PSD Template</title>
+    <link href="css/style.css" rel="stylesheet">
+    <link href="http://allfont.ru/allfont.css?fonts=montserrat" rel="stylesheet" type="text/css" />
+</head>
+<body>
+    <div id="mySidenav" class="default-hidden sidenav">
+        <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
```

Comment:
Semantically, it's better to wrap all a tags in li and ul
ul.sidenav__list > li.sidenav__list-item > a 

P.S. Class naming style is described here: [Yandex BEM](https://en.bem.info/methodology/)